### PR TITLE
Work around spec issue

### PR DESCRIPTION
**Pull request by jfbastien**
https://github.com/WebAssembly/spec/issues/248 is causing the waterfall to fail. Work around the issue.

### CODE REVIEW

File: src/build.py [hunk 1]
```diff
@@ -261,6 +261,10 @@ def GitCloneFetchCheckout(self):
     """Clone a git repo if not already cloned, then fetch and checkout."""
     if os.path.isdir(self.src_dir):
       print '%s directory already exists' % self.name
+      if self.name == 'spec':
+        # TODO(jfb): Work around https://github.com/WebAssembly/spec/issues/248
+        proc.check_call(['git', 'checkout', '--', 'ml-proto/winmake.bat'],
```
Comment (dschuff) [inline src/build.py]: How about just git reset --hard?

Comment (jfbastien) [inline src/build.py]: I don't want to clobber local changes... well, except for this one ;-)

Comment (dschuff) [inline src/build.py]: Or better yet:
git clean -dffx
git reset --hard HEAD

see https://chromium.googlesource.com/native_client/src/native_client/+/master/pynacl/repo_tools.py#172

Comment (jfbastien) [inline src/build.py]: I just want to work around that one thing, without resetting everything else.

**Pull request merged**